kythe/cxx/indexer/cxx/testdata/basic/macros_lambda_call.cc

```diff
@@ -0,0 +1,12 @@
+#define CHECK(x) (x)
+
+//- @Call defines/binding CallDecl
```

Comment:
Can you add some non-verifier comments to this file, documenting what it's checking for?

Reply:
Shoot, missed this. I can do that in a follow up.